addons/payment/models/payment_icon.py

```diff
@@ -8,12 +8,12 @@ class PaymentIcon(models.Model):
     _description = 'Payment Icon'
     _order = 'sequence, name'
 
-    name = fields.Char(string="Name")
+    name = fields.Char(string="Name", required=True)
     provider_ids = fields.Many2many(
         string="Providers", comodel_name='payment.provider',
         help="The list of providers supporting this payment icon")
     image = fields.Image(
-        string="Image", max_width=64, max_height=64,
+        string="Image", max_width=64, max_height=64, required=True,
```

Comment:
A bit of nitpicking :) If all the parameters don't fit on one line, we now tend to put one on each line.
```suggestion
        string="Image", 
        max_width=64, 
        max_height=64, 
        required=True,
```

Reply:
Done :)